src/test/java/io/vertx/test/codegen/testapi/jsonmapper/WithMyCustomEnumWithMapper.java

```diff
@@ -0,0 +1,27 @@
+package io.vertx.test.codegen.testapi.jsonmapper;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import io.vertx.codegen.annotations.GenIgnore;
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+
+@VertxGen
```

Comment:
I don't think `@VertxGen` is needed here

Reply:
Yep, it is... at least I received a test failure after removed it.

Reply:
sorry for that